monai/transforms/utils_create_transform_ims.py

```diff
@@ -0,0 +1,220 @@
+    """
+    Find the documentation for a given transform and if it's missing,
+    add a pointer to the transform's example image.
+    """
+    with open(code_path) as f:
+        contents = f.readlines()
+    doc_start = None
+    for i, line in enumerate(contents):
+        # find the line containing start of the transform documentation
+        if "`" + transform_name + "`" in line:
+            doc_start = i
+            break
+    if doc_start is None:
+        raise RuntimeError("Couldn't find transform documentation")
+
+    # if image is already in docs, nothing to do
+    image_line = doc_start + 2
+    if ".. image" in contents[image_line]:
+        return
+
+    # add the line for the image and the alt text
+    contents_orig = deepcopy(contents)
+    contents.insert(
+        image_line,
+        ".. image:: https://github.com/Project-MONAI/DocImages/raw/main/transforms/" + transform_name + ".png\n",
+    )
+    contents.insert(image_line + 1, "    :alt: example of " + transform_name + "\n")
+
+    # check that we've only added two lines
+    assert len(contents) == len(contents_orig) + 2
+
+    # write the updated doc to overwrite the original
+    with open(code_path, "w") as f:
+        f.writelines(contents)
+
+
+def pre_process_data(data, ndim, is_map):
+    """If transform requires 2D data, then convert to 2D"""
+    if ndim == 2:
+        for k in KEYS:
+            data[k] = data[k][..., data[k].shape[-1] // 2]
+
+    return data if is_map else data[CommonKeys.IMAGE]
+
+
+def get_2d_slice(image, view):
+    """Get the central slice of a 3D volume"""
+    shape = image.shape
+    slices = [slice(0, s) for s in shape]
+    _slice = shape[view] // 2
+    slices[view] = slice(_slice, _slice + 1)
+    slices = tuple(slices)
+    return np.squeeze(image[slices], view)
+
+
+def get_stacked_2d_ims(im):
+    """Get the 3 orthogonal views and stack them into 1 image.
+    Requires that all images be same size, but this is taken care
+    of by the `SpatialPadd` earlier.
+    """
+    return np.hstack([get_2d_slice(im, view) for view in range(3)])
+
+
+def get_stacked_before_after(before, after):
+    """Stack before and after images into 1 image.
+    Requires that before and after images be the same size.
+    """
+    return np.vstack([get_stacked_2d_ims(d[0]) for d in (before, after)])
+
+
+def save_image(images, labels, filename):
+    """Save image to file, ensuring there's no whitespace around the edge."""
+    sizes = images.shape
+    fig = plt.figure()
+    fig.set_size_inches(1.0 * sizes[1] / sizes[0], 1, forward=False)
+    ax = plt.Axes(fig, [0.0, 0.0, 1.0, 1.0])
+    ax.set_axis_off()
+    fig.add_axes(ax)
+    ax.imshow(images, cmap="gray")
+    if labels is not None:
+        ax.imshow(labels, cmap="hsv", alpha=0.9)
+    fig.savefig(filename, dpi=images.shape[0])
+    plt.close(fig)
+
+
+def create_transform_im(transform, data, ndim, update_doc=True, out_dir=None, seed=0):
```

Comment:
@SachidanandAlle mentioned that providing this utility to the users so that they can visualise the effects of the transforms for any input images. Looks like a nice feature to have cc @rijobro @Nic-Ma @ericspod